packages/core/src/types.ts

```diff
@@ -1,26 +1,7 @@
-/**
- * The below few interfaces (until end comment) should be defined in the @stoplight/types repo
- */
-
-export interface IDisposable {
-  dispose: () => void;
-}
-
+import { IDiagnostic, Omit } from '@stoplight/types';
+export type INoRangeDiagnostic = Omit<IDiagnostic, 'range'>;
 export type Path = Array<string | number>;
```

Comment:
`JsonPath` is exported by `@stoplight/types`

Reply:
This type was not even used. 😱